src/template.ts

```diff
@@ -13,6 +13,7 @@ Number of annotations:: {{annotations.length}}
 - 🔖 Context:: {{#if contextualText}}{{{contextualText}}}{{else}}N/A{{/if}}
 - 🎯 Highlight:: {{{highlight}}}
 - 📝 Note:: {{#if note}}{{{note}}}{{else}}N/A{{/if}}
+- 📙 Highlight Link:: {{#if highlightLocation}}[Apple Books Highlight Link](ibooks://assetid/{{../bookId}}#{{highlightLocation}}){{else}}N/A{{/if}}
```

Comment:
1. I believe we can remove the `if` statement because there cannot be a highlight without location.
2. What is the purpose of `../` in `{{../bookId}}` ? Shouldn't it be like in the example you specified in the README? Seems like Handlebars ignores this "path-like" construction and just inserts the variable value instead, like it should.

```suggestion
- 📙 Highlight Link:: [Apple Books Highlight Link](ibooks://assetid/{{bookId}}#{{highlightLocation}})
```

Reply:
You're welcome, thank you for your feedback.

1. I thought it would be a good fail-safe in case for some reason the highlight is available but the location is not (e.g. a corrupt database or some other edge case). If the location is guaranteed to exist if the highlight is present then I agree we should remove it

2. Well spotted. The reason for using `{{../bookId}}` is because when we're inside an `#each` block in handlebars, `bookId` is not in scope. However, we can use `../` path segment to reference the parent template scope. See [Access a variable outside the scope of a Handlebars.js each loop](https://stackoverflow.com/questions/13645084/access-a-variable-outside-the-scope-of-a-handlebars-js-each-loop)]

I'll test point 2 locally and if I can still access `{{bookId}}` without `../` path syntax then we can simply use `{{bookId}}`, however if it doesn't work then I'll update the README to with `{{../bookId}}`

Let me know your thoughts on 1 and 2 and then I'll proceed to make the changes.

Reply:
@absorpheus Thank you for the explanation.
1. Got it, it does make sense as a precaution. I believe that a corrupted database is more of a rare edge case, when information won't be available at all rather than partially, but we can check it later.
2. That's a good thing to keep in mind, actually, and maybe it also worth documenting in future as a troubleshooting point.